src/types/common.ts

```diff
@@ -1,3 +1,5 @@
 export interface IModal {
 	setShowModal: (showModal: boolean) => void;
+	continueProcess?: () => void;
+	type?: string;
```

Comment:
please add these to your specific interface, we should keep the base interface abstract.